### PR TITLE
Dev env fixes

### PR DESCRIPTION
**Pull request by ohel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,45 @@
...
+
+More detailed documentation is found in a README in respective directories of each project.
+
+## Prerequisites
 
 - Docker
-- Node JS - v10
-- PostgreSQL - v11
+- NodeJS - v10
+- PostgreSQL - v11 (probably works with v10 and v12 also)
```
Comment (vinujosep) [inline README.md:14]: haven't tested with v10, but tested with v12 locally and it didn't work

File: frontend/README.md [hunk 2]
```diff
@@ -1,44 +1,52 @@
...
+**`eject`**
 
 **Note: this is a one-way operation. Once you `eject`, you can’t go back!**
 
 If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
 
 Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+You never need to use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
```
Comment (vinujosep) [inline frontend/README.md:45]: As we are cleaning up the boilerplate READMEs with what is only needed, do we really need to add these?

Comment (vinujosep) [reply frontend/README.md:45]: the same for admin-frontend

Comment (ohel) [reply frontend/README.md:45]: Yeah I was wondering about the same, but then again, ":D". At least the script descriptions are good to have.

**Comment by ohel** [comment]
> General question: should we be using both package-lock.json and yarn.lock for /frontend

No we shouldn't. Some frontend dev may resolve the issue at some point.

**Pull request merged**